pkg/koordlet/metrics/common.go

```diff
@@ -65,6 +65,12 @@ var (
 		Help:      "Number of cpu cores used by node in realtime",
 	}, []string{NodeKey})
 
+	ReportNodeMetricStatus = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
we need to define a rule for new collector/qos-strategy that what a new plugin should exposed for observability, e.g. rule parse failure, dependency status

Reply:
added a basic metric for health status of modules and plugins, more detailed metrics will be added in another PR